services/docsrs/docsrs.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.array()
+  .items(
+    Joi.object({
+      build_status: Joi.boolean().required(),
+    })
+  )
+  .min(1)
+  .required()
+
+module.exports = class DocsRs extends BaseJsonService {
+  static category = 'build'
+  static route = { base: 'docsrs', pattern: ':crate/:version?' }
+  static examples = [
+    {
+      title: 'docs.rs',
+      namedParams: { crate: 'regex', version: 'latest' },
+      staticPreview: this.render({ version: 'latest', buildStatus: true }),
+      keywords: ['rust'],
+    },
+  ]
+
+  static defaultBadgeData = {}
+
+  static render({ buildStatus, version }) {
+    if (buildStatus) {
+      return {
+        label: `docs@${version}`,
```

Comment:
I think we'll probably want to eschew this in cases where `version` is `latest`, similar to how we don't have ![](https://img.shields.io/badge/build@latest-passing-brightgreen) because the absence of a specific version implicitly conveys latest/current status.